examples/runner-public/main.tf

```diff
@@ -19,13 +19,14 @@ module "runner" {
   aws_region  = "${var.aws_region}"
   environment = "${var.environment}"
 
-  ssh_public_key = "${local_file.public_ssh_key.content}"
+  ssh_key_name = "${local.key_pair_name}"
```

Comment:
Should refer the the terraform resource not to the local var to avoid this resource (launch group) is created before the keypair

Reply:
I think this is good now thanks to your other commit(s)